tools/build.js

```diff
@@ -50,7 +50,7 @@ const getBuildOptions = input => {
     );
     const type = ['classic'];
     const mapTypeToSource = {
-        classic: './code/es-modules',
+        classic: [output, 'es-modules'].join(''),
         css: './code/js/es-modules'
```

Comment:
Two thoughts here:
- `path.join` from `require('path')` is the proper way to connect path fragments
- Not related to your work, so you do not need to fix it. I just notice that the css path makes no sense. There is no `code/js` folder structure anywhere